pkg/generated/crds/manifests/backupdriver.io_backuprepositories.yaml

```diff
@@ -0,0 +1,58 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (devel)
+  creationTimestamp: null
+  name: backuprepositories.backupdriver.io
+spec:
+  group: backupdriver.io
+  names:
+    kind: BackupRepository
+    listKind: BackupRepositoryList
+    plural: backuprepositories
+    singular: backuprepository
+  scope: Namespaced
```

Comment:
Should BackupRepository be cluster-scoped?

Reply:
Yes.  Fixed.